packages/pnpm/test/recursive/misc.ts

```diff
@@ -360,6 +361,53 @@ test('recursive installation of packages with hooks', async t => {
   t.ok(lockfile2.packages['/dep-of-pkg-with-1-dep/100.1.0'])
 })
 
+test('recursive installation of packages in workspace ignores hooks in package', async t => {
+  // This test hangs on Appveyor for some reason
+  if (isCI && isWindows()) return
```

Comment:
I copied the framework for these tests from a test that had this check; I'm unsure if it'll apply to the tests I wrote to I put it in one and not the other.